src/ui/components/RecommendedBadge/index.js

```diff
@@ -16,31 +17,43 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
+  _window?: typeof window | Object,
   i18n: I18nType,
 |};
 
-export const RecommendedBadgeBase = ({ i18n, size }: InternalProps) => {
+export const sumoLink = 'https://support.mozilla.org/';
+
+export const RecommendedBadgeBase = ({
+  _window = typeof window !== 'undefined' ? window : {},
+  i18n,
+  size,
+}: InternalProps) => {
   const label = i18n.gettext('Recommended');
 
+  const openSumoLink = (event: SyntheticEvent<HTMLButtonElement>) => {
+    _window.open(sumoLink, '_blank');
```

Comment:
I was wondering how one could fix the issue with a `<button />`. I have mixed feelings about this _hack_. Ideally, we should fix the issue so that the links works without JavaScript. I'd suggest that we follow what is described here: https://www.sarasoueidan.com/blog/nested-links/.

That would fix another issue I noticed while testing this patch: the button does not look like a link, it shows the default cursor and not the pointer.

I think that will be a little more work, but that would be more future-proof (I believe). WDYT?

Reply:
That's an interesting article. I had read about the `<object>` hack and didn't like the looks of it. I will try implementing what she suggests in the article and see if I can get it to work in a satisfactory manner.

Reply:
@willdurand I was trying to understand exactly what she is doing in the article, but I don't think I quite do. I'm having trouble getting any of it to work. I tried the second to last example, which seems to be syntactically correct, but doesn't do what is described. Then I looked at the final example, and I cannot make sense of it as it contains invalid style syntax (e.g., `top: ...;`). So I'm kind of stuck. Do you think you could take a stab at making it work, or even provide me with a simple example that _does_ work?